litho-it/src/test/java/com/facebook/litho/widget/TextSpecTest.java

```diff
@@ -38,11 +38,15 @@
 import com.facebook.litho.testing.eventhandler.EventHandlerTestHelper;
 import com.facebook.litho.testing.helper.ComponentTestHelper;
 import com.facebook.litho.testing.testrunner.ComponentsTestRunner;
+import com.facebook.yoga.YogaDirection;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.RuntimeEnvironment;
 
+import javax.annotation.Nullable;
```

Comment:
Can you, please, use AndroidX annotation here, so that we won't need to import LITHO_JSR_TARGET as a dep in BUCK?

Reply:
lol you're gonna have to ask @astreet for these fixes now :P

Reply:
Yeah, it's ok. We pulled this in order to test in asap :)